Allow files in same directory to be loaded without prefixing './'

ndwarshuis opened the pull request.

s-zeng commented on dhall/__init__.py:
```diff
@@ -16,5 +16,7 @@ def remember_cwd():
-        os.chdir(os.path.dirname(fp.name))
+        newdir = os.path.dirname(fp.name)
+        if newdir != "":
+            os.chdir()
```
Looks like you just need to put in the newdir argument and you should be good.

s-zeng reviewed (changes requested):
Thanks for contributing! Let's get the test passing again, and maybe add a test as well, and we should be good to go

ndwarshuis commented:
Fixed typo and also added a new test (and the tests should actually pass). I just reused the already-present test file for the new test and called it in a slightly different order; hopefully that's clear enough.

ndwarshuis commented:
@s-zeng were you still waiting on me to do something more with this?

ndwarshuis commented:
thanks :)
